From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-hwpoison-coredump-support-recovery-from-dump_user_range-checkpatch-fixes
Date: Thu Apr 13 03:05:23 PM PDT 2023

WARNING: 'an user' may be misspelled - perhaps 'a user'?
#33: 
address may not always an user address, so introduce a new copy_mc flag in
                       ^^^^^^^

WARNING: please, no spaces at the start of a line
#133: FILE: lib/iov_iter.c:637:
+       if (iov_iter_is_copy_mc(i))$

WARNING: suspect code indent for conditional statements (7, 15)
#133: FILE: lib/iov_iter.c:637:
+       if (iov_iter_is_copy_mc(i))
+               return (void *)copy_mc_to_kernel(to, from, size);

ERROR: code indent should use tabs where possible
#134: FILE: lib/iov_iter.c:638:
+               return (void *)copy_mc_to_kernel(to, from, size);$

WARNING: please, no spaces at the start of a line
#134: FILE: lib/iov_iter.c:638:
+               return (void *)copy_mc_to_kernel(to, from, size);$

total: 1 errors, 4 warnings, 118 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

NOTE: Whitespace errors detected.
      You may wish to use scripts/cleanpatch or scripts/cleanfile

./patches/mm-hwpoison-coredump-support-recovery-from-dump_user_range.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: Kefeng Wang <wangkefeng.wang@huawei.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/iov_iter.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/lib/iov_iter.c~mm-hwpoison-coredump-support-recovery-from-dump_user_range-checkpatch-fixes
+++ a/lib/iov_iter.c
@@ -634,8 +634,8 @@ EXPORT_SYMBOL_GPL(_copy_mc_to_iter);
 static void *memcpy_from_iter(struct iov_iter *i, void *to, const void *from,
 				 size_t size)
 {
-       if (iov_iter_is_copy_mc(i))
-               return (void *)copy_mc_to_kernel(to, from, size);
+	if (iov_iter_is_copy_mc(i))
+		return (void *)copy_mc_to_kernel(to, from, size);
 	return memcpy(to, from, size);
 }
 
_
